pkgs/applications/blockchains/dogecoin/default.nix

```
@@ -1,12 +1,14 @@
 { lib, stdenv , fetchFromGitHub
 , pkg-config, autoreconfHook
 , db5, openssl, boost, zlib, miniupnpc, libevent
-, protobuf, util-linux, qt4, qrencode
-, withGui }:
+, protobuf, qtbase ? null
+, wrapQtAppsHook ? null, qttools, qmake ? null, qrencode
+, withGui, withUpnp ? true, withUtils ? true, withWallet ? true
+, withZmq ? true, zeromq, util-linux ? null, Cocoa ? null }:
```

```suggestion
, withZmq ? true, zeromq, util-linux, Cocoa }:
```
if those are null something is broken.

---

Not required arguments on darwin